Xamarin.Forms.Core.UnitTests/AbsoluteLayoutTests.cs

```diff
@@ -174,7 +174,7 @@ public void SizeRequestWithRelativeChild ()
 
 			var sizeReq = abs.GetSizeRequest (double.PositiveInfinity, double.PositiveInfinity);
 
-			Assert.AreEqual (new Size (200, 40), sizeReq.Request);
+			Assert.AreEqual (new Size (100, 20), sizeReq.Request);
```

Comment:
what was wrong with this test ?

Reply:
When calculating `SizeRequest` no longer divided the size of child element with proportional width/height.